ogc/wmts-layer/README.md

```diff
@@ -1,23 +1,35 @@
-# WMTS Layer
+# WMTS layer
 
-Dsiplay tiles from a Web Map Tile Service.
+Display a layer from a Web Map Tile Service.
+ 
+![Image of WMTS layer](WmtsLayer.png)
 
-![](WmtsLayer.png)
+## Use case
 
-## How it works
+WMTS services can have several layers. You can use Runtime to explore the layers available from a service. This would commonly be used to enable a browsing experience where users can choose which layers they want to display at run time.
```

Comment:
```suggestion
WMTS services can have several layers. You can use ArcGIS Runtime to explore the layers available from a service. This would commonly be used to enable a browsing experience where users can choose which layers they want to display at run time.
```